dev/custom-pages/academics-search.scss

```diff
@@ -316,4 +315,76 @@
       }
     }
   }
+}
+
+#contentcol .resources-section {
+  padding-bottom: 35px;
+}
+
+#contentcol .callout {
+  background: #fc0;
+  margin: 0 auto;
+  text-align: center;
+  padding: 48px 0;
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  .container {
+    vertical-align: top;
+  }
+
+  .icon-link {
+    display: inline-block;
+    margin: 10px auto;
+    width: 25%;
+    vertical-align: top;
+
+    @media (max-width: $screen-xs-max) {
+      width: 50%;
+    }
+
+    a {
+      @include gotham();
+      color: #000;
+
+      .fa {
+        font-size: 80px;
+        margin-bottom: 12px;
+        @include transition(color .25s);
+      }
+
+      h3 {
+        @include gotham();
+        background: rgba(30, 30, 30, 0.25);
+        border-radius: 4px;
+        display: inline;
+        font-size: 14px;
+        padding: 2px 14px;
+        @include transition(background .25s, color .25s);
+      }
+
+      p {
+        font-size: 12px;
+        font-style: italic;
+        padding-top: 8px;
+      }
+
+      &:hover,
```

Comment:
More of a style preference, but: can these nested state selectors be moved up above the other nested element/classes, since they modify the parent selector?  Not a critical thing, but helps in terms of organization.

Example: https://www.viget.com/articles/organizing-nested-selectors-in-sass